plugins/mcreator-core/datalists/stepsounds.yaml

```diff
@@ -217,4 +217,10 @@
 - SPONGE:
   texture: SPONGE#0
 - WET_SPONGE:
-  texture: SPONGE#1
\ No newline at end of file
+  texture: SPONGE#1
+- VAULT
+  texture: VAULT
+- HEAVY_CORE
+  texture: HEAVY_CORE
+- COBWEB
+  texture: COBWEB
```

Comment:
Texture for COBWEB is called WEB so please fix this

![image](https://github.com/MCreator/MCreator/assets/16374228/15abb4a5-9061-4dc7-84cb-3bf5d8adf3c3)